### DISCUSSION TITLE
Bridge for Wyze Cam

### DISCUSSION THREAD

**Discussion by petercxc** [tteck/Proxmox#1470] [Ideas]
is there any hope you will develope lxc script for this project? [WebRTC/RTSP/RTMP/HLS Bridge for Wyze Cam]
(https://github.com/mrlt8/docker-wyze-bridge)
Thanks!

**Comment by mgrimace** [comment]
This along with Scrypted with OpenCV + prebuffer + homekit addons will turn a Wyze camera into a Homekit Secure Video Device. I have both Docker Wyze Bridge, and Scrypted running in the Home Assistant OS LXC.
